dir.csv

```diff
@@ -0,0 +1,6 @@
+molecule,resname,path
```

Comment:
needs to be somewhere inside the tests, not top level

Reply:
I think this was an accident from running tests, or was somehow saved during testing of the workflows base module and not removed as a temporary file. I will look into it.